apps/prairielearn/elements/pl-string-input/pl-string-input.py

```diff
@@ -207,9 +216,21 @@ def parse(element_html: str, data: pl.QuestionData) -> None:
         data["submitted_answers"][name] = None
         return
 
+    # Do unicode decode
     if normalize_to_ascii:
         a_sub = unidecode(a_sub)
```

Comment:
Should this migrate to the new `pl.full_unidecode`?

Reply:
Ah yep, good catch.

Reply:
Done here: https://github.com/PrairieLearn/PrairieLearn/pull/8670/commits/b3f245e8bf53739ad258c11a040627b46b1deb52